src/combo_container.ts

```diff
@@ -71,11 +71,18 @@ class ComboContainer {
       }
     };
 
+    this.comboEditor.onComboClick = (c, idx) => {
+      this.delete(`${c}${idx}`);
+      this.update();
+    }
+
     this.boardWidthStatus = () => parseInt(this.comboEditor.boardWidthInput.value);
     this.comboEditor.boardWidthInput.onchange = () => {
       this.update();
     }
 
+    // what does this do? It looks like comboEditor.getInputElements always
```

Comment:
This code was how a user interacted with the combos before having visible combos.  This, however, is no longer necessary since I've removed that functionality.

Reply:
Got it, I'll remove it.